### PR TITLE
Feature/delete pet

### PR DESCRIPTION
**Pull request by apete12**
What I did: Add delete pet functionality with associated styling changes. 

Testing Notes/Extras: 

Did this break anything?
- [X]  No
- [ ]  Yes

Type of change
- [X]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- []  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
Add testing for delete pet functionality.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -28,14 +41,13 @@ const App = () => {
...
 
-  console.log(allPets)
 
   return (
     <div className="App">
       <Header />
       {loading && <Loading />}
       <Routes>
-        <Route path="/" element={<HomeView addNewPet={addNewPet} allPets={allPets} setLoading={setLoading} setError={setError}/>}/>
+        <Route path="/" element={<HomeView addNewPet={addNewPet} deletePet={deletePet} allPets={allPets} setLoading={setLoading} setError={setError}/>}/>
```
Comment (judy0ye) [inline src/App.js:50]: thanks for adding the delete!

File: src/components/PetCard/PetCard.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
 import './PetCard.css';
 import { Link } from 'react-router-dom';
 import { checkIcon, checkLinkImage } from '../../utils';
+import deleteIcon from '../../images/delete.png';
+import spirals from '../../images/spirals.png';
```
Comment (judy0ye) [inline src/components/PetCard/PetCard.js:5]: love the spirals!

**Review by judy0ye — APPROVED** [review]

File: src/components/AllPets/AllPets.js
Comment (paulina-isabel) [inline src/components/AllPets/AllPets.js:1]: Yay! Looks way cleaner.

**Review by paulina-isabel — APPROVED** [review]
Great work.

**Review by aliceabarca — APPROVED** [review]
great job adding the delete feature.

**Pull request merged**